### ISSUE TITLE
Question: Is this repo still alive and used?

### ISSUE THREAD

**Issue by volekerb** [CityOfZion/neo-sharp#573]
Browsing through several projects related to Neo nodes and it seems that this repo not used anymore since 2018 there are no new issues and in https://hub.docker.com/r/cityofzion/neo-sharp no images uploaded.
I believe it will be good to keep one single point of entry for Neo node build with C# so it's easier to get in and start contributing for newcomers.

Suggestions:
1. Add reference to the new repo(I think it's https://github.com/neo-project/neo-node correct me if I'm wrong)  used for C# node and archive this one (probably)
2. Update https://hub.docker.com/r/cityofzion/neo-sharp (remove or add at least one single neo sharp docker image which could be used as a single node)

**Comment by osmirnov** [comment]
Yes, it is an inactive project. For the latest development please refer to [neo](https://github.com/neo-project/neo) and [neo-node](https://github.com/neo-project/neo-node).
